sidebars.js

```diff
@@ -89,6 +89,11 @@ module.exports = {
           label: 'Prime',
```

Comment:
Question: Should this say Prime V1 now?

Reply:
Yeah that's a good ideal.

Reply:
I ended up making the Prime V1 link point to the /v1 url now. https://github.com/transcom/mymove-docs/pull/383/commits/9a54bcfcfcfa3fdf2d65da0ad0e7f79dc4d72b8d